### PR TITLE
OHRM5X-192: Add cypress testing framework

### PR DESCRIPTION
**Pull request by samanthajayasinghe**

### CODE REVIEW

File: tests/Readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## Intrdocution
```
Comment (RajithaKumara) [inline tests/Readme.md:1]: Typo: Introduction

File: tests/cypress.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "baseUrl": "http://php80/orangehrm/symfony/web/index.php",
```
Comment (RajithaKumara) [inline tests/cypress.json:2]: Shall we add this config change to readme

File: tests/cypress/fixtures/user.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        "userName": "John",
+        "password": "John@123",
+        "fullName": "John Perera"
+    },
+    "mike": {
+        "userName": "Mike",
+        "password": "Mike@123",
+        "fullName": "Mike Combas"
+    }
+}
```
Comment (RajithaKumara) [inline tests/cypress/fixtures/user.json:17]: Can you add new line

File: tests/Readme.md (hunk 1 above)
Comment (samanthajayasinghe) [reply tests/Readme.md]: done

File: tests/cypress.json (hunk 2 above)
Comment (samanthajayasinghe) [reply tests/cypress.json:2]: done

File: tests/cypress/fixtures/user.json (hunk 3 above)
Comment (samanthajayasinghe) [reply tests/cypress/fixtures/user.json]: done

**Pull request merged**